[2021-12-14 08:05:20,210] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: postgres_operator_dag.create_pet_table manual__2021-12-14T08:05:19.119259+00:00 [queued]>
[2021-12-14 08:05:20,222] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: postgres_operator_dag.create_pet_table manual__2021-12-14T08:05:19.119259+00:00 [queued]>
[2021-12-14 08:05:20,222] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-14 08:05:20,223] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-12-14 08:05:20,223] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-14 08:05:20,237] {taskinstance.py:1262} INFO - Executing <Task(PostgresOperator): create_pet_table> on 2021-12-14 08:05:19.119259+00:00
[2021-12-14 08:05:20,242] {standard_task_runner.py:52} INFO - Started process 781 to run task
[2021-12-14 08:05:20,246] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_operator_dag', 'create_pet_table', 'manual__2021-12-14T08:05:19.119259+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/pets_dag.py', '--cfg-path', '/tmp/tmpy_ekdaz4', '--error-file', '/tmp/tmpwcju4kdg']
[2021-12-14 08:05:20,247] {standard_task_runner.py:77} INFO - Job 24: Subtask create_pet_table
[2021-12-14 08:05:20,302] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_operator_dag.create_pet_table manual__2021-12-14T08:05:19.119259+00:00 [running]> on host 4ee0ea791fc7
[2021-12-14 08:05:20,374] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgres_operator_dag
AIRFLOW_CTX_TASK_ID=create_pet_table
AIRFLOW_CTX_EXECUTION_DATE=2021-12-14T08:05:19.119259+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-14T08:05:19.119259+00:00
[2021-12-14 08:05:20,384] {base.py:79} INFO - Using connection to: id: ***_conn_id. Host: postgres-***, Port: 5432, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-14 08:05:20,388] {dbapi.py:225} INFO - Running statement: -- create hello_world_table
DROP TABLE IF EXISTS pet;
CREATE TABLE pet (
    pet_id SERIAL PRIMARY KEY,
    name VARCHAR NOT NULL,
    pet_type VARCHAR NOT NULL,
    birth_date DATE NOT NULL,
    OWNER VARCHAR NOT NULL
    );, parameters: None
[2021-12-14 08:05:20,419] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=postgres_operator_dag, task_id=create_pet_table, execution_date=20211214T080519, start_date=20211214T080520, end_date=20211214T080520
[2021-12-14 08:05:20,459] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-14 08:05:20,508] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
